packages/eslint-plugin/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://nx.dev",
   "peerDependencies": {
-    "@typescript-eslint/parser": "^6.13.2",
+    "@typescript-eslint/parser": "^7.0.0",
```

Comment:
this should probably allow both `^6.13.2` and `^7.0.0`

Reply:
Yeah, done